dspace/config/item-submission.xml

```diff
@@ -7,219 +7,275 @@
+            <heading>submit.progressbar.describe.stepone</heading>
+            <processing-class>org.dspace.app.rest.submit.step.DescribeStep</processing-class>
+            <type>submission-form</type>
+        </step>
+        <step id="journalVolumeStep" mandatory="true">
+            <heading>submit.progressbar.describe.stepone</heading>
+            <processing-class>org.dspace.app.rest.submit.step.DescribeStep</processing-class>
+            <type>submission-form</type>
+        </step>
+        <step id="journalIssueStep" mandatory="true">
+            <heading>submit.progressbar.describe.stepone</heading>
+            <processing-class>org.dspace.app.rest.submit.step.DescribeStep</processing-class>
+            <type>submission-form</type>
+        </step>
+
+        <step id="upload">
+            <heading>submit.progressbar.upload</heading>
+            <processing-class>org.dspace.app.rest.submit.step.UploadStep</processing-class>
+            <type>upload</type>
+        </step>
+        <step id="license">
+            <heading>submit.progressbar.license</heading>
+            <processing-class>org.dspace.app.rest.submit.step.LicenseStep</processing-class>
+            <type>license</type>
+            <scope visibilityOutside="read-only">submission</scope>
+        </step>
+
+        <!-- OpenAIRE submission steps/forms -->
+        <step id="openAIREProjectForm" mandatory="true">
+            <heading>submit.progressbar.describe.stepone</heading>
+            <processing-class>org.dspace.app.rest.submit.step.DescribeStep</processing-class>
+            <type>submission-form</type>
+        </step>
+
+        <step id="openAIREPersonForm" mandatory="true">
+            <heading>submit.progressbar.describe.stepone</heading>
+            <processing-class>org.dspace.app.rest.submit.step.DescribeStep</processing-class>
+            <type>submission-form</type>
+        </step>
+        <step id="openAIREOrganizationForm" mandatory="true">
+            <heading>submit.progressbar.describe.stepone</heading>
+            <processing-class>org.dspace.app.rest.submit.step.DescribeStep</processing-class>
+            <type>submission-form</type>
+        </step>
+        <step id="openAIREPublicationPageoneForm" mandatory="true">
+            <heading>submit.progressbar.describe.stepone</heading>
+            <processing-class>org.dspace.app.rest.submit.step.DescribeStep</processing-class>
+            <type>submission-form</type>
+        </step>
+        <step id="openAIREPublicationPagetwoForm" mandatory="true">
+            <heading>submit.progressbar.describe.stepone</heading>
+            <processing-class>org.dspace.app.rest.submit.step.DescribeStep</processing-class>
+            <type>submission-form</type>
+        </step>
+
+        <!-- Step Upload Item with Embargo Features to enable this step, please 
+            make sure to comment-out the previous step "UploadStep" <step id="upload-with-embargo"> 
```

Comment:
Because some of these commented out steps reference other steps (see this comment), I'd prefer it if we move *all* the OpenAIRE specific `<step>` definitions to the bottom of the `</step-definitions>` section.   Inserting them in the middle (where they currently are) seems awkward, as some of the commented out steps below are often used in local configurations.

Reply:
I tried to do that as much as possible (having OpenAIRE4 stuff at the end). What is happening here is related with the original file indentation. Which is based on TABs and my Eclipse it's configured to use only Spaces (as referred in DSpace Code Style Guidelines - https://wiki.lyrasis.org/display/DSPACE/Code+Style+Guide) so, formatting source files, the TABs are replaced. That's why all these changes. But if you check the source file - https://github.com/DSpace/DSpace/blob/4c914c147b40d9d8d13f2a313e1268041918a67e/dspace/config/item-submission.xml you will understand that we didn't change it.

Reply:
@paulo-graca : I think you misunderstood my comment here.  I'm noting that I think the OpenAIRE4 stuff should be at the *very end* of the `</step-definitions>` section.  Currently, it is *not* at the very end...instead there are a number of commented out step definitions after the OpenAIRE4 stuff.  This causes oddities like above...notice that the line here says to
> make sure to comment-out the previous step "UploadStep"
However, now the previous step is *not* "UploadStep", instead it's the "openAIREPublicationPagetwoForm" step.

So, my recommendation is to move all the OpenAIRE4 steps to the very bottom of the list, just before the "sample" step.

Reply:
Ok, I've just move them to the line just before this:
`        <!-- This is the Sample Step which utilizes the JSPSampleStep class -->
        <step id="sample">
            <heading>Sample</heading>
            <processing-class>org.dspace.submit.step.SampleStep</processing-class>
            <type>sample</type>
        </step>`